packages/flutter_blue/example/tizen/tizen-manifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.example.flutter_blue_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="mobile"/>
+    <ui-application appid="com.example.flutter_blue_tizen_example" exec="runner" type="capp" multiple="false" nodisplay="false" taskmanage="true" hw-acceleration="on">
```

Comment:
What's the reason for using the capp type?

Reply:
It just turned out to be easier to debug. No problem in changing it back to `C#` if that's preferred.

Reply:
Yes, native apps are easier to debug, but please use the `dotnet` type in the final code. That's the convention of this repository because native apps are not generally allowed to run on TV.